package.json

```diff
@@ -14,16 +14,16 @@
   "scripts": {
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha -r test/fixtures/env.js test/**/*.js",
     "start": "node index",
+    "start-dev": "npm run watch & npm run start",
```

Comment:
This only runs the webpack watch and blocks, and then never runs the lounge itself.